### DISCUSSION TITLE
Journaling features

### DISCUSSION THREAD

**Discussion by ecoutinho** [netbox-community/netbox#6205] [Ideas]
I was using the "Comments" section to keep a record of hardware failures and thought it would be a good idea to transfer this information into the new "Journal" section. While doing this, I've noticed two limitations of the journal implementation:

- it doesn't allow to change the date of the entry. For the purpose of documenting past events it would be nice to record the date it occurred and not the date I enter this information. Of course I can write the date on the text box, but it looks messy.
- global search is not searching on this field. It would be nice if it did, as it does on "Comments".

Thanks.

**Comment by proudbro** [comment]
Yes, it would be very useful to change the date of journal entry for planning maintenance
